### ISSUE TITLE
Line number in the syntax error message in CI

### ISSUE THREAD

**Issue by jiwoongim** [sbdchd/squawk#365]
Is there a way to output the line number when it gives a syntax error?

For example, I get 
```  
note: Postgres failed to parse query: syntax error at or near "("
help: Modify your Postgres statement to use valid syntax.
``` 
it is hard to find which line the syntax error. comes from.

**Comment by sbdchd** [comment]
Yeah definitely possible to add, the libpg_query-sys crate has:

```rs
#[repr(C)]
#[derive(Debug, Copy, Clone)]
pub struct PgQueryError {
    pub message: *mut ::std::os::raw::c_char,
    pub funcname: *mut ::std::os::raw::c_char,
    pub filename: *mut ::std::os::raw::c_char,
    pub lineno: ::std::os::raw::c_int,
    pub cursorpos: ::std::os::raw::c_int,
    pub context: *mut ::std::os::raw::c_char,
}
```

but we only use the `message` field from it:

https://github.com/sbdchd/squawk/blob/ca1f346d4a98529418a0ee635efe4e11c5580a86/parser/src/parse.rs#L26-L27

**Comment by sbdchd** [comment]
Should be fixed my squawk version 2.0.0, which now includes a new error reseliant parser

for example:


```sh
echo 'select 1 2 3' | squawk
warning[syntax-error]: missing comma
 --> stdin:1:9
  |
1 | select 1 2 3
  |         ^
  |
warning[syntax-error]: missing comma
 --> stdin:1:11
  |
1 | select 1 2 3
  |           ^
  |

Find detailed examples and solutions for each rule at https://squawkhq.com/docs/rules
Found 2 issues in 1 file (checked 1 source file)
```
